projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -1,24 +1,45 @@
 <ng-container *ngIf="order$ | async as order">
   <!-- consigned entries -->
-  <ng-container *ngIf="orderPromotions$ | async as orderPromotions">
-    <cx-promotions [promotions]="orderPromotions"></cx-promotions>
-  </ng-container>
+  <ng-container *ngIf="order.consignments?.length">
+    <ng-container *ngIf="orderPromotions$ | async as orderPromotions">
+      <cx-promotions [promotions]="orderPromotions"></cx-promotions>
```

Comment:
we need to have this once as there's the duplicate in the unconsigned entries part

it would like this

**- display promotion**
- display status
-display consigned entries if they have (in the order that we have right now)
- display unconsigned entries at the bottom


but we have this right now 
**- display promotion**
- display status
-display consigned entries if they have (in the order that we have right now)
**- display promotion**
- display unconsigned entries at the bottom


* we need @Xymmer  to get back to us on this on Monday

Reply:
Moved order promotions to the top of the template and added the proper conditionals